packages/near-membrane-base/src/membrane.ts

```diff
@@ -112,13 +112,19 @@ export type HooksCallback = (
 
 export type DistortionCallback = (target: ProxyTarget) => ProxyTarget;
 
+interface InstrumentationHooks {
+    start(label: string): void;
```

Comment:
@garychangsf the label (which is probably the wrong term) should probably be one of the two values needed for the instrumentation to be useful, e.g.: `inbound` and `SetPrototypeOf` are more useful IMO.

Reply:
@dbajric we might need your input here, what other information you might need here?

Reply:
the other problem that I have identified here is the fact that in blue we don't know who the `red` is (which namespace it is), that might be problematic and might require some extra arguments to the blue-red analogy.

Reply:
we can pass in both inbound/outbound and the name of the operation.

Reply:
namespace would be nice and give us a little bit clues directly from telemetry data. But probably unnecessary for now.